stdlib/testing/fail.go

```diff
@@ -0,0 +1,22 @@
+package testing
+
+import (
+	"context"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+)
+
+func init() {
+	flux.RegisterPackageValue("testing", "fail", values.NewFunction(
```

Comment:
I would prefer not adding this function to the stdlib.

Reply:
Is there a reason? It seems like a reasonably useful function since we use it in a lot of packages. Would you feel more comfortable with it if I threw it into an `internal/testing` package instead?

Reply:
From my knowledge `fail` is used only for tests where we want the flux script to produce an error if something unexpected happens. It doesn't seem like a useful function to expose publicly. Especially since you can achieve the equivalent functionality in a single line of pure flux `fail = () => [][0]`

Reply:
I threw the function into `internal/testutil`. I like it because you could eventually customize the error message. If you prefer the implementation to be pure flux, I can change it to that. An internal package means that we don't have to decide if it is stdlib worthy or not and I still can access it from the tests without booting up the repl.